src/test/java/org/folio/rest/api/PatronNoticePoliciesApiTest.java

```diff
@@ -209,12 +218,37 @@ public void cannotDeleteNonexistentPatronNoticePolicyId() throws MalformedURLExc
     assertThat(response.getBody(), is(NOT_FOUND));
   }
 
+  @Test
+  public void cannotDeleteInUsePatronNoticePolicy() throws InterruptedException, MalformedURLException,
+    TimeoutException, ExecutionException {
+
+    String inUsePolicyId = "16b88363-0d93-464a-967a-ad5ad0f9187c";
```

Comment:
Why is this hard coded, rather than generated during the test?

Reply:
I prefer to use hard coded values rather then generated in tests, because it makes test debugging easier. Do you think such approach has some disadvantages?

Reply:
I think it is a less representative example of actual usage and I think it makes it more likely that the test will pass/fail for the wrong reason (e.g. this test could be made to pass with a special case for this ID)

Others have also remarked that they feel hard coded IDs are easier for debugging. I think that is a signal that either the test or the system aren't providing sufficient diagnostic information (and there are likely plenty of cases of this). 

These are only my thoughts, I don't think this isn't something important enough to block merging.